data/lang/quitconfirmation-core/en.json

```diff
@@ -0,0 +1,82 @@
+{
+   "QUIT" : {
+      "description" : "Quit game, as a question",
+      "message" : "Quit?"
+   },
+   "YES" : {
+      "description" : "Yes, quit program",
+      "message" : "Yes"
+   },
+   "NO" : {
+      "description" : "No, don't quit program",
+      "message" : "No"
+   },
+   "MSG_1" : {
+      "description" : "",
+      "message" : "Get out of here and go back to your boring programs?"
+   },
+   "MSG_2" : {
+      "description" : "",
+      "message" : "Don't go now, there's a Pirate waiting for you in the other program."
+   },
+   "MSG_3" : {
+      "description" : "",
+      "message" : "Are you sure you want to quit this great game that cost you nothing?"
+   },
+   "MSG_4" : {
+      "description" : "",
+      "message" : "Doest thou wish to leave with such hasty abandon?"
+   },
+   "MSG_5" : {
+      "description" : "(Inspired by Wolfenstein 3D)",
+      "message" : "Choose {no} for guns and glory, {yes} for work and worry."
```

Comment:
Ironically, I'm thinking both yes and no.

Uppercase does have its merits, but in other missions we have `{price}`, not `{PRICE}`, so just trying to be consistent.

If that was what you meant?